study.md

```diff
@@ -49,15 +53,16 @@ should contain two strings, "drinking beer" and "conspiracy theories".  The keys
 should be symbols.
 
 ```ruby
-#your answer here
+dale["middle name"] = "Alvin"
```

Comment:
Almost.  This will create a string key named `middle name`.  There needs to be a colon to denote that this is a symbol.

```ruby
some_hash = {}
some_hash['some key'] = 'asdf'
some_hash[:'some key'] = 'zxcv'
some_hash # => {"some key"=>"asdf", :"some key"=>"zxcv"}
```